src/main/java/edu/kis/powp/jobs2d/command/manager/CommandBoundaryCheckObserver.java

```diff
@@ -0,0 +1,30 @@
+package edu.kis.powp.jobs2d.command.manager;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.BoundaryCheckJob2dDriver;
+import edu.kis.powp.jobs2d.features.DriverFeature;
+import edu.kis.powp.observer.Subscriber;
+
+import java.util.logging.Logger;
+
+public class CommandBoundaryCheckObserver implements Subscriber {
+    private final CommandManager commandManager;
+
+    private Logger logger = Logger.getLogger(Logger.GLOBAL_LOGGER_NAME);
+
+    public CommandBoundaryCheckObserver(CommandManager commandManager) {
+        this.commandManager = commandManager;
+        this.commandManager.getChangePublisher().addSubscriber(this);
+    }
+
+    @Override
+    public void update() {
+        Job2dDriver currentDriver = DriverFeature.getDriverManager().getCurrentDriver();
+        BoundaryCheckJob2dDriver boundaryCheckDriver = new BoundaryCheckJob2dDriver(currentDriver);
```

Comment:
Now loading a command results not only in BoundaryCheck, but also in executing it using currentDriver. It is not a desired behavior. This is an unnecessary mixing of responsibilities.